fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java

```diff
@@ -88,17 +91,25 @@ public class CatalogMgr implements Writable, GsonPostProcessable {
     private final Map<Long, CatalogIf> idToCatalog = Maps.newConcurrentMap();
     // this map will be regenerated from idToCatalog, so not need to persist.
     private final Map<String, CatalogIf> nameToCatalog = Maps.newConcurrentMap();
+
+    private static final Map<String, Integer[]> refreshMap = Maps.newConcurrentMap();
```

Comment:
Add some comment in code to explain the key value in refreshMap

Reply:
ok,I will add some comment